cmd/admin-main.go

```diff
@@ -43,7 +43,7 @@ var adminCmd = cli.Command{
 }
 
 func mainAdminBefore(ctx *cli.Context) error {
-	color.Yellow("\t *** Warning:Experiment feature. Not Ready for Production ***")
+	color.Yellow("\t *** mc admin heal is EXPERIMENTAL ***")
```

Comment:
This message will be shown in all admin commands and not only with heal. I am not sure we want for all admin commands or only for heal.